Source/Scenes/Details/XDetailsNode.swift

```diff
@@ -241,14 +248,14 @@ class XDetailsNode: ASCellNode {
             actionButton.isEnabled = displayOptions.enablePlaceHold
         }
         Style.styleButton(asInverse: actionButton)
-        Style.setButtonTitle(actionButton, title: actionButtonText, fontSize: 15)
+        Style.setButtonTitle(actionButton, title: actionButtonText, fontSize: subtitleSize)
```

Comment:
I really like where this is going and almost all of the changes seem solid.  The one I question is using `subtitleSize` here.   That makes these buttons larger than buttons on other screens which are 17pt.  I would rather use `bodySize` here for now and make the buttons 44pt using a different mechanic.  I'll suggest one below.